### PR TITLE
dataset: beijing sentence corpus

### PR DESCRIPTION
**Pull request by SiQube**

### CODE REVIEW

File: tests/unit/datasets/datasets_test.py [hunk 1]
```diff
@@ -31,6 +31,7 @@
     ('public_dataset', 'dataset_name'),
     # XXX: add public dataset in alphabetical order
```
Comment (dkrako) [inline tests/unit/datasets/datasets_test.py:32]: this comment here should probably removed, right? It was introduced by https://github.com/aeye-lab/pymovements/commit/054d567fe5c5e9b8f42c2ee193eb5a8b61f3c993

File: tests/unit/datasets/datasets_test.py
Comment (dkrako) [inline tests/unit/datasets/datasets_test.py:1]: There are still tests missing in `tests/functional` right?

File: tests/unit/datasets/datasets_test.py (hunk 1 above)
Comment (SiQube) [reply tests/unit/datasets/datasets_test.py:32]: I think we can leave it but maybe remove the `XXX`?

File: tests/unit/datasets/datasets_test.py
Comment (SiQube) [reply tests/unit/datasets/datasets_test.py:1]: we don't have too much functionality for pre-computed events hence no tests (yet) added

File: tests/unit/datasets/datasets_test.py (hunk 1 above)
Comment (dkrako) [reply tests/unit/datasets/datasets_test.py:32]: # please add datasets in alphabetical order

File: tests/unit/datasets/datasets_test.py
Comment (dkrako) [reply tests/unit/datasets/datasets_test.py:1]: ah right, didn't see that it's an event-only dataset

**Review by dkrako — APPROVED** [review]
Great! Thanks a lot for increasing the size of our dataset library!

**Pull request merged**